purview/utils/general.py

```diff
@@ -169,10 +169,9 @@ def create_query(item, uid_name, uid, reverse=False):
 
     if query:
         query += """\
-            \'}) YIELD path
+            \', minLevel:1}) YIELD path
             RETURN path
             ORDER BY length(path) DESC
-            LIMIT 1
```

Comment:
Make this a kwarg so that `get_story` doesn't get all the paths returned by Neo4j. It's just unnecessary churn.